app/javascript/src/components/GameLogEntry.vue

```diff
@@ -1,17 +1,132 @@
 <template>
-  <div class="border border-black rounded p-2 m-2">
+  <div v-if="Array.isArray(event)" class="border border-black rounded p-2 m-2 rondel">
+    <div v-for="(action, index) in event" :key="(index + Math.random()).toString()">
+      <div v-if="action.type === 'rondel'">
+        <div>
+          <svg 
+            class="inline-block mr-1" 
+            xmlns="http://www.w3.org/2000/svg" 
+            width="30" 
+            height="20"
+            >
+            <Flag :nation="action.payload.nation.value" :width="(30).toString()"></Flag>
+          </svg>{{ processAction(action) }}
+        </div>
+      </div>  
+      <div v-else>
+        <div>
+          - {{ processAction(action) }}
+        </div>
+      </div>
+    </div>
+  </div>
+  <div v-else>
     <div>
-      <b>Type:</b> {{ action.type }}
+      <b>Type:</b> {{ event.type }}
     </div>
     <div>
-      <b>Payload:</b> {{ action.payload }}
+      <b>Payload:</b> {{ event.payload }}
     </div>
-  </div>
+    <div>
+      <b>Text:</b> {{ processAction(event) }}
+    </div>
+  </div> 
 </template>
 
 <script>
+import Flag from "./flags/Flag.vue";
+import stringify from "../stringify.js";
 export default {
   name: "GameLogEntry",
-  props: { action: Object }
+  props: { events: Object, index: Number },
+  computed: {
+    event: function() {
+      return this.events.event
+    }
+  },
+  components: { Flag },
+  methods: {
+    capitalize(word) {
+      if (word) {
+      return (word[0].toUpperCase() + word.substring(1));
+      } else {
+        return "";
+      }
+    },
+    processAction(action) {
+      let notImplemented = "NOT IMPLEMENTED";
+      switch (action.type) {
+        case "initialize":
+          return notImplemented;
```

Comment:
Can we implement this now or would you rather wait until later. Either way is totally ok with me!

Reply:
Sure, what is the text pattern that you wanted for this?